packages/forma-36-react-components/src/components/Button/README.mdx

```diff
@@ -26,55 +26,92 @@ Buttons communicate the action that will occur when the user clicks them. They c
 
 ## Component variations
 
+### Types
+
 - **Primary** - Used for the most important actions in any scenario. Don’t use more than one primary button in a section or screen to avoid overwhelming users.
 - **Muted** - Used for a secondary actions, the most commonly used button type.
 - **Positive** - For use when the action has a positive connotation such as creating or publishing a new entity.
 - **Negative** - For destructive actions - when something can't be undone. For example, deleting entities.
-- **Naked** - Link button
+- **Warning** - (deprecated)
+- **Naked** - (deprecated)
 
 ```jsx
 <>
-  <Button buttonType="primary">Primary</Button>
-  <Button buttonType="negative">Negative</Button>
-  <Button buttonType="positive">Positive</Button>
-  <Button buttonType="muted">Muted</Button>
-  <Button buttonType="naked">Naked</Button>
+  <Flex flexDirection="row" marginBottom="spacingM">
+    <Grid
```

Comment:
I think this example is too complicated and it steals attention from the main goal of the example

Reply:
We just need a new layout component I guess: https://chakra-ui.com/docs/layout/stack

Reply:
How do you suggest to fix the issue reported then?
Inline css?

Reply:
I prefer not using Chakra and try to promote our Library
Since we do not have a component like Chakra’s `Stack`, how should I layout it? Or should we keep the example without any layout?

Reply:
yeah, sure. I mean, that we should implement a similar component in v4 :) 

1. I'd wrap each button in a div with a margin.
2. or I'd leave it as it is. It's already fixed in v4 branch if I'm not mistaken

Reply:
It does not seem to be fixed
https://github.com/contentful/forma-36/blob/forma-v4/packages/components/button/README.mdx

It's the same example that it was before

![Screenshot 2021-08-09 at 17 48 25](https://user-images.githubusercontent.com/6597467/128735095-476d17ae-d749-45c9-89a5-23d8b1a9e20b.png)

I will refactor it and use just css then

Reply:
I see, then we should probably wrap all buttons in a ButtonGroup variant="spaced" in v4 branch.

Reply:
Do you think this would be too distracting?

```jsx
<Grid rowGap="spacingXs" rows="repeat(4, auto)" justifyContent="start">
  <Button buttonType="primary">Primary</Button>
  <Button buttonType="muted">Muted</Button>
  <Button buttonType="positive">Positive</Button>
  <Button buttonType="negative">Negative</Button>
</Grid>
```

It's basically Chakra’s VStack example that you posted before

Reply:
I think it's better than before. We can go with it.